### PR TITLE
implements feat 151: student work image gallery

### LINKED ISSUE #151: render examples of student work in activity view
- [x] add to activity model in contentful
- [x] make an activity component that renders images in a gallery
- [x] create thumbnail gallery in image-gallery component
- [x] implement photoswipe in image-gallery component
  - http://codepen.io/dimsemenov/pen/ZYbPJM
  - http://photoswipe.com/documentation/getting-started.html

### PR DESCRIPTION
**Pull request by eemshi**
closes https://github.com/codefordenver/owlet/issues/151

### CODE REVIEW

File: resources/public/js/vendor/initPhotoSwipeFromDOM.js [hunk 1]
```diff
@@ -0,0 +1,202 @@
+var initPhotoSwipeFromDOM = function(gallerySelector) {
```
Comment (dviramontes) [inline resources/public/js/vendor/initPhotoSwipeFromDOM.js:1]: nice!, its own file. Thank you!

**Review by dviramontes — APPROVED** [review]
bravo 👏 
🚢 it :)

**Comment by dviramontes** [comment]
Also @eemshi do we have a separate issue for considering deep-linking to images ?
The `gid=1&pid=5` url params we disabled.

**Comment by eemshi** [comment]
@dviramontes i didn't disable the url params this time, but it's difficult to test the deep-links because of the activity-view loading issue. i'll make a note to re-test once we merge your loading fixes

**Pull request merged**